packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -371,6 +378,59 @@ export class AndroidEmulatorDevice extends DeviceBase {
     ]);
   }
 
+  async mirrorNativeLogs(build: AndroidBuildResult) {
+    const startTime = strftime("%F %T.000", new Date());
+
+    if (this.nativeLogsCancelToken) {
+      this.nativeLogsCancelToken.cancel();
+    }
+
+    this.nativeLogsCancelToken = new CancelToken();
+
+    const extractPidFromLogcat = async (cancelToken: CancelToken) =>
+      new Promise<string>((resolve, reject) => {
+        const regexString = `Start proc ([0-9]{4}):${build.packageName}`;
+        const process = exec(ADB_PATH, ["logcat", "-e", regexString, "-T", startTime]);
```

Comment:
We need the `-T` param, otherwise, it will catch the same pid (when we restart the app) - the first one

Reply:
`-T` also takes a number, so maybe we could do `-T 1` to not read old logs? (`-T 0` is not legal)